app/state_machines/efile_submission_state_machine.rb

```diff
@@ -21,8 +21,8 @@ class EfileSubmissionStateMachine
   transition from: :queued,       to: [:transmitted, :failed, :rejected]
   transition from: :transmitted,  to: [:accepted, :rejected]
 
-  guard_transition(to: :queued) do |submission|
-    submission.submission_bundle.present?
+  guard_transition(to: :queued) do |submission, transition|
+    transition.metadata[:seeding].present? || submission.submission_bundle.present?
```

Comment:
this allows people to run `rake efile:seed_submissions` without running into errors. Feels weird though. may remove guard totally and just handle the error in the queuing Job